openpype/hosts/maya/plugins/publish/extract_camera_mayaScene.py

```diff
@@ -158,14 +160,14 @@ def process(self, instance):
                             frame_range=[start, end],
                             step=step
                         )
-                        baked_camera_shapes = cmds.ls(baked,
-                                               type="camera",
-                                               dag=True,
-                                               shapes=True,
-                                               long=True)
-
-                        members = members + baked_camera_shapes
-                        members.remove(camera)
+                        baked_camera_shapes = set(cmds.ls(baked,
+                                                  type="camera",
+                                                  dag=True,
+                                                  shapes=True,
+                                                  long=True))
+
+                        members = members.union(baked_camera_shapes)
```

Comment:
No need to return a new set if you're assigning it to the same variable anyway.
```suggestion
                        members.update(baked_camera_shapes)
```